Reporting/M5 Document/Status.md

```diff
@@ -0,0 +1,123 @@
+### **[Status of the software implementation]{.underline}**
+
+-   [How many of your initial requirements that your team set out to
+    > deliver did you actually deliver (a checklist/table would help to
+    > summarize)? Were you able to deliver everything or are there
+    > things missing? Did your initial requirements sufficiently capture
+    > the details needed for the project?]{.underline}
+
++---------------------------------------------------------+------------+
+| **Initial Requirements**                                | **D        |
+|                                                         | elivered** |
++=========================================================+============+
+| Allow users to create an account on the platform.       | -          |
++---------------------------------------------------------+------------+
+| Allow the user to log in to the platform with an        | -          |
+| existing account.                                       |            |
++---------------------------------------------------------+------------+
+| Allow the user to log out of their account.             | -          |
++---------------------------------------------------------+------------+
+| Allow the user to edit their account (username,         | -          |
+| password, email, etc.).                                 |            |
++---------------------------------------------------------+------------+
+| Allow the user to send text messages.                   | -          |
++---------------------------------------------------------+------------+
+| Allow the user to receive text messages.                | -          |
++---------------------------------------------------------+------------+
+| Allow users to search for other users on the platform.  | -          |
++---------------------------------------------------------+------------+
+| Allow the user to add another user as a friend.         | -          |
++---------------------------------------------------------+------------+
+| Allow users to remove another user from their friend    | -          |
+| list.                                                   |            |
++---------------------------------------------------------+------------+
+| Allow the user to create a group chat.                  | -          |
```

Comment:
Change to "to create a text channel". Users can only create text channels as DMs are created when a friend request is accepted by default. That means the "new text channel" modal only servers to create a channel having at least 3 people in it (DMs are not allowed as they come from your friend list).